apps/nextjs/src/components/board/items/item-actions.tsx

```diff
@@ -62,7 +61,7 @@ export const useItemActions = () => {
           options: {},
           width: 1,
           height: 1,
-          integrations: [],
+          integrationIds: [],
```

Comment:
Technically this type could be defined since we export all the possible integration IDs

Reply:
No as integrationIds are not the kinds and rather the primary keys from the database